p2p/p2ptest/network.go

```diff
@@ -34,16 +34,24 @@ func MakeNetwork(t *testing.T, nodes int) *Network {
 		logger:        logger,
 		memoryNetwork: p2p.NewMemoryNetwork(logger),
 	}
+
 	for i := 0; i < nodes; i++ {
 		node := MakeNode(t, network)
 		network.Nodes[node.NodeID] = node
 	}
 
+	return network
+}
+
+// Start starts the network by setting up a list of node addresses to dial in
+// addition to creating a peer update subscription for each node. Finally, all
+// nodes are connected to each other.
+func (n *Network) Start(t *testing.T) {
```

Comment:
note to reviewer: `Start` was added because in reactor tests, we want to fully _create and start_ the reactor before getting full peer connectivity.